index.html

```diff
@@ -68,6 +68,128 @@ <h2>I'am a Software Developer</h2>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Data Dashboard Healthcare</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="tags">
+                <a href="#">html</a>
+                <a href="#">bootstrap</a>
+                <a href="#">Ruby</a>
+              </div>
+            </div>
+            <button>See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Website Portfolio</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="tags">
+                <a href="#">html</a>
+                <a href="#">bootstrap</a>
+                <a href="#">Ruby</a>
+              </div>
+            </div>
+            <button>See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Profesional Art Printing Data More</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="tags">
+                <a href="#">html</a>
+                <a href="#">bootstrap</a>
+                <a href="#">Ruby</a>
+              </div>
+            </div>
+            <button>See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Data Dashboard Healthcare</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="tags">
+                <a href="#">html</a>
+                <a href="#">bootstrap</a>
+                <a href="#">Ruby</a>
+              </div>
+            </div>
+            <button>See Project</button>
+          </div>
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Website Portfolio</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <div class="tags">
+                <a href="#">html</a>
+                <a href="#">bootstrap</a>
+                <a href="#">Ruby</a>
+              </div>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+      </section>
     </main>
```

Comment:
Kindly add the aboutme section as well 📝

Reply:
Will do! @mateo951